run.py

```diff
@@ -0,0 +1,27 @@
+"""
+Usage:
+- Run app: python run.py
```

Comment:
this is sweet, any way we could automate this in some way? maybe a github action that makes a commit? It's a little annoying/risky for automated commits but I feel like it makes sense for docs, idk if @fantix has an opinion